public/js/controllers/home-controller.js

```diff
@@ -120,4 +121,71 @@ function($scope, $http, planService, notificationService) {
             });
         });
     };
+
+    $scope.hasPrereqs = function(course) {
+
+        //This is ugly
```

Comment:
In this case the comments are telling a story that is explaining what is happening and there is no reason to toggle them as comments or not. If they were code snippets that you would have to uncomment for debugging, then use the "//" that the keybindings give you, though these generally won't be in our master/production branch and we shouldn't worry about this case. I just think that the /* */ are easier to read for block comments, especially ones that are supposed to be explaining things in normal language.